### PR TITLE
get orchestrator compiling again

### PR DESCRIPTION
**Pull request by levicook**
These are the manual edits we had to make to get gravity_proto to compile.

### CODE REVIEW

**Review by marbar3778 — APPROVED** [review]
should we merge this now and then we can take a look at what else is missing

File: orchestrator/gravity_utils/src/connection_prep.rs [hunk 1]
```diff
@@ -236,7 +236,7 @@ pub async fn check_delegate_addresses(
     prefix: &str,
 ) {
     let eth_response = client
-        .get_delegate_key_by_eth(QueryDelegateKeysByEthAddress {
+        .delegate_keys_by_validator(DelegateKeysByValidatorAddress {
```
Comment (levicook) [inline orchestrator/gravity_utils/src/connection_prep.rs:239]: @jackzampolin I wasn't 100% sure about this rename; mind confirming I picked correctly on this?

**Comment by levicook** [comment]
Down to 15 compiler errors within the gravity_utils crate.  May need to pair up with someone on the Erc20Tokens vs sdk.Coin stuff.

```bash
   Compiling gravity_utils v0.1.0 (orchestrator/gravity_utils)
error[E0609]: no field `orchestrator_address` on type `DelegateKeysByOrchestratorAddressResponse`
   --> gravity_utils/src/connection_prep.rs:255:58
    |
255 |             let req_delegate_eth_address: EthAddress = o.orchestrator_address.parse().unwrap();
    |                                                          ^^^^^^^^^^^^^^^^^^^^ unknown field
    |
    = note: available fields are: `validator_address`, `ethereum_signer`

error[E0609]: no field `validator_address` on type `DelegateKeysByValidatorAddressResponse`
   --> gravity_utils/src/connection_prep.rs:288:18
    |
288 |             if e.validator_address != o.validator_address {
    |                  ^^^^^^^^^^^^^^^^^ unknown field
    |
    = note: available fields are: `eth_address`, `orchestrator_address`

error[E0308]: mismatched types
  --> gravity_utils/src/types/batches.rs:31:49
   |
31 |             erc20_token: Erc20Token::from_proto(input.erc20_token.unwrap())?,
   |                                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `gravity_proto::gravity::Erc20Token`, found struct `gravity_proto::cosmos_sdk_proto::cosmos::base::v1beta1::Coin`

error[E0308]: mismatched types
  --> gravity_utils/src/types/batches.rs:32:47
   |
32 |             erc20_fee: Erc20Token::from_proto(input.erc20_fee.unwrap())?,
   |                                               ^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `gravity_proto::gravity::Erc20Token`, found struct `gravity_proto::cosmos_sdk_proto::cosmos::base::v1beta1::Coin`

error[E0308]: mismatched types
  --> gravity_utils/src/types/batches.rs:74:51
   |
74 |             let tx = BatchTransaction::from_proto(tx)?;
   |                                                   ^^ expected struct `SendToEthereumTx`, found struct `Vec`
   |
   = note: expected struct `SendToEthereumTx`
              found struct `Vec<u8>`

error[E0609]: no field `orchestrator` on type `BatchTxSignature`
   --> gravity_utils/src/types/batches.rs:117:33
    |
117 |             orchestrator: input.orchestrator.parse()?,
    |                                 ^^^^^^^^^^^^ unknown field
    |
    = note: available fields are: `token_contract`, `nonce`, `eth_signer`, `signature`

error[E0599]: no method named `parse` found for struct `Vec<u8>` in the current scope
   --> gravity_utils/src/types/batches.rs:120:44
    |
120 |             eth_signature: input.signature.parse()?,
    |                                            ^^^^^ method not found in `Vec<u8>`

error[E0609]: no field `transfers` on type `ContractCallTx`
  --> gravity_utils/src/types/logic_call.rs:25:31
   |
25 |         for transfer in input.transfers {
   |                               ^^^^^^^^^ unknown field
   |
   = note: available fields are: `invalidation_nonce`, `invalidation_scope`, `contract_call_address`, `payload`, `timeout` ... and 2 others

error[E0609]: no field `contract` on type `gravity_proto::cosmos_sdk_proto::cosmos::base::v1beta1::Coin`
  --> gravity_utils/src/types/logic_call.rs:34:45
   |
34 |                 token_contract_address: fee.contract.parse()?,
   |                                             ^^^^^^^^ unknown field
   |
   = note: available fields are: `denom`, `amount`

error[E0609]: no field `invalidation_id` on type `ContractCallTx`
  --> gravity_utils/src/types/logic_call.rs:49:36
   |
49 |             invalidation_id: input.invalidation_id,
   |                                    ^^^^^^^^^^^^^^^ help: a field with a similar name exists: `invalidation_nonce`

error[E0308]: mismatched types
  --> gravity_utils/src/types/logic_call.rs:70:47
   |
70 |             invalidation_id: hex_str_to_bytes(&input.invalidation_id).unwrap(),
   |                                               ^^^^^^^^^^^^^^^^^^^^^^ expected `str`, found struct `Vec`
   |
   = note: expected reference `&str`
              found reference `&Vec<u8>`

error[E0609]: no field `orchestrator` on type `ContractCallTxSignature`
  --> gravity_utils/src/types/logic_call.rs:72:33
   |
72 |             orchestrator: input.orchestrator.parse()?,
   |                                 ^^^^^^^^^^^^ unknown field
   |
   = note: available fields are: `invalidation_id`, `invalidation_nonce`, `eth_signer`, `signature`

error[E0599]: no method named `parse` found for struct `Vec<u8>` in the current scope
  --> gravity_utils/src/types/logic_call.rs:74:44
   |
74 |             eth_signature: input.signature.parse()?,
   |                                            ^^^^^ method not found in `Vec<u8>`

error[E0609]: no field `orchestrator` on type `UpdateSignerSetTxSignature`
  --> gravity_utils/src/types/valsets.rs:67:33
   |
67 |             orchestrator: input.orchestrator.parse()?,
   |                                 ^^^^^^^^^^^^ unknown field
   |
   = note: available fields are: `nonce`, `eth_signer`, `signature`

error[E0599]: no method named `parse` found for struct `Vec<u8>` in the current scope
  --> gravity_utils/src/types/valsets.rs:70:44
   |
70 |             eth_signature: input.signature.parse()?,
   |                                            ^^^^^ method not found in `Vec<u8>`

error: aborting due to 15 previous errors

Some errors have detailed explanations: E0308, E0599, E0609.
For more information about an error, try `rustc --explain E0308`.
error: could not compile `gravity_utils`
```

File: orchestrator/gravity_utils/src/connection_prep.rs [hunk 2]
```diff
@@ -285,7 +285,7 @@ pub async fn check_delegate_addresses(
                 exit(1);
             }
 
-            if e.validator_address != o.validator_address {
+            if e.orchestrator_address != o.validator_address {
```
Comment (levicook) [inline orchestrator/gravity_utils/src/connection_prep.rs:288]: I'm not 100% confident in this

**Comment by levicook** [comment]
Closing this --- everything here has been pulled on to PR4 https://github.com/PeggyJV/gravity-bridge/pull/4

**Pull request closed without merging**